### PR TITLE
upgrade to ember-cli-0.1.15 and ember 1.18

### PR DESCRIPTION
**Pull request by boazin**

### CODE REVIEW

File: webapp/Brocfile.js [hunk 1]
```diff
@@ -2,7 +2,11 @@
 
 var EmberApp = require('ember-cli/lib/broccoli/ember-app');
 
-var app = new EmberApp();
+var app = new EmberApp(/*{
+  vendorFiles: {
```
Comment (vmalloc) [inline webapp/Brocfile.js]: Why is this commented out?

Comment (boazin) [inline webapp/Brocfile.js]: Was getting "Uncaught Error: Could not find module handlebars".
From what I understood the change will be needed with Handlebars 2.0

I didn't notice I left it commented. I'll delete it

Comment (boazin) [inline webapp/Brocfile.js]: fixed

**Comment by vmalloc** [comment]
ember 1.18? I think you meant 1.10...

**Comment by boazin** [comment]
oops... 1.8.1

**Comment by vmalloc** [comment]
Ok :-)

**Pull request merged**